### DISCUSSION TITLE
Connecting Phoenix Server to Supabase DB in Replit

### DISCUSSION THREAD

**Discussion by aar2dee2** [supabase/supabase#3157] [Questions] [answered]
I'm trying to connect my Phoenix (Elixir web dev framework) hosted on Replit to a Supabase DB.
This is what my config/dev.exs looks like:
<img width="562" alt="Screenshot 2021-09-11 at 12 19 47 PM" src="https://user-images.githubusercontent.com/85004512/132939393-85181f7c-1875-4502-8f59-3c212b6839b4.png">

I'm getting a 'non existing domain' error when I run `mix ecto.create` in the project directory
<img width="587" alt="Screenshot 2021-09-11 at 12 22 39 PM" src="https://user-images.githubusercontent.com/85004512/132939437-7d30b850-34ee-4236-80c4-191f52494675.png">

Any ideas on what's not working?

**Comment by silentworks** [comment, marked as answer]
The hostname is incorrect in this instance. It should only be `db.tcdwhkvbpgytprftchhr.supabase.co`. I think the rest is constructed from the credentials you entered in your config file.

**Reply by aar2dee2** [reply]
Wow, that worked! thanks a ton!!
